File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/Java/7_Queue.js
import java.util.LinkedList;
import java.util.Queue;
public class QueueInJava {
    public static void main(String[] args) {
        //Tao 1 Queue kieu int
        Queue<Integer> myQueue = new LinkedList<>();

        //2. add cac value vao Queue moi tao
        myQueue.add(1);
        myQueue.add(2);
        myQueue.add(3);

        System.out.print("myQueue: ");
        while(!myQueue.isEmpty()){
            System.out.print(myQueue.peek()+ " "); //ktra p.tu dau cua Queue la value nao
            myQueue.remove(); //loai bo p.tu dau
        }

    }
}

File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/C++/7_Queue.cpp
#include <iostream>
#include <queue>
using namespace std;

int main()
{
    //Tao 1 queue kieu int
    queue<int> myQueue;

    //2. add cac value vao queue moi tao
    myQueue.push(1);
    myQueue.push(2);
    myQueue.push(3);

    cout << "myQueue: ";
    while(!myQueue.empty()){
        cout << myQueue.front() << " "; //ktra p.tu dau cua queue la value nao
        myQueue.pop(); //loai bo p.tu dau
    }
}

